FIXED: Before this PR, when using a custom view with subviews, subviews were presented in the reversed order

didierkorthoudt opened the pull request:
Workaround for the ibtool bug which reverses subviews order when using a custom view.
This is valid as of ibtool version 14460.31

mrcarlberg commented:
As this is a trick to fix a bug in `ibtools` why not just do `_subviews.reverse();` instead of this manual reverse?

Also `setSubviews:` does a lot of things and is that necessary here?

didierkorthoudt commented:
@mrcarlberg OK, I've implemented your suggestion and it works without needing ```setSubview:``` & Co. as ```nib2cib``` doesn't cope with the DOM. I've also added a manual test to verify that it works as expected.

Before the fix : 
![Bad](https://user-images.githubusercontent.com/2394110/93716045-643dcc00-fb6d-11ea-9ecd-6603ca7c8799.png)

After the fix : 
![Good](https://user-images.githubusercontent.com/2394110/93716047-6acc4380-fb6d-11ea-90a7-ac87cb3c9342.png)

mrcarlberg commented:
Merged! Thanks!
